engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -107,10 +108,10 @@ object WorkflowActor {
     * @param currentLifecycleStateActor The current lifecycle stage, represented by an ActorRef.
     */
   case class WorkflowActorData(currentLifecycleStateActor: Option[ActorRef],
-                               workflowDescriptor: Option[EngineWorkflowDescriptor])
+                               workflowDescriptor: Option[EngineWorkflowDescriptor],
+                               executionResponse: Option[WorkflowExecutionActorResponse])
   object WorkflowActorData {
-    def empty = WorkflowActorData(None, None)
-    def apply(currentLifecycleStateActor: ActorRef, workflowDescriptor: EngineWorkflowDescriptor): WorkflowActorData = WorkflowActorData(Option(currentLifecycleStateActor), Option(workflowDescriptor))
+    def empty = WorkflowActorData(None, None, None)
```

Comment:
totally thinking out loud here... maybe this would be more clear as named parameters or default parameter values on the case class